test/data/manifests/fedora-coreos-container.mpp.yaml

```diff
@@ -602,3 +602,17 @@ pipelines:
             type: qcow2
             compression: false
             compat: '1.1'
+  - name: zip
+    build: name:build
+    stages:
+      - type: org.osbuild.zip
+        inputs:
+          tree:
+            type: org.osbuild.files
+            origin: org.osbuild.pipeline
+            references:
+              name:tree:
+                file: /ostree/repo
+        options:
+          filename: disk.zip
+          paths: ['/ostree/repo/state', '/ostree/repo/config']
```

Comment:
rather than update the fedora-coreos-container.mpp.yaml here it might be better to just creat a new zip specific test like the one for tar in https://github.com/osbuild/osbuild/tree/main/test/data/stages/tar